lr-cli/cortx_setup/commands/node/prepare/firewall.py

```diff
@@ -0,0 +1,72 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+from ...command import Command
+from provisioner.salt import (
+    local_minion_id, 
+    StatesApplier
+)
+from provisioner.config import (
+    CORTX_CONFIG_DIR,
+    PRVSNR_PILLAR_DIR
+)
+from provisioner.errors import (
+    ReleaseFileNotFoundError
+)
+from provisioner.utils import dump_yaml
+from pathlib import Path
+import yaml
+
+class NodePrepareFirewall(Command):
+    _args = {
+        'config': {
+            'type': str,
+            'optional': True}
+    }
+
+    def run(self, **kwargs):
+
+        node_id = local_minion_id()
+        firewall_pillar_sls = Path(f"{PRVSNR_PILLAR_DIR}/components/firewall.sls")
+
+        self.logger.info(f"updating firewall config on {node_id}")
+
+        try:
+            self.logger.info(f"loading firewall configuration")
+            firewall_config_arg = kwargs.get('config')
+
+            if firewall_config_arg.startswith("yaml://"):
+                firewall_config_dir = Path(firewall_config_arg.split('//')[1])
+
+                if firewall_config_dir.exists() :
+                    with open(firewall_config_dir) as file:
+                        data = yaml.load(file, Loader=yaml.FullLoader)
```

Comment:
can we use confstore to load and store values to yaml files ?

Doc for confstore : https://seagatetechnology.sharepoint.com/:w:/r/sites/gteamdrv1/tdrive1224/_layouts/15/doc2.aspx?sourcedoc=%7BA68B2DB0-B041-4136-8F51-94F0A4685D1E%7D&file=ConfStore_Specification.docx&action=default&mobileredirect=true&cid=1962a4af-69b3-4075-bf25-d172eda3e4f8